iOS/rainbow/View/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -149,30 +149,6 @@
       "idiom" : "ios-marketing",
       "filename" : "ItunesArtwork@2x.png",
       "scale" : "1x"
-    },
```

Comment:
I am not sure why these were deleted. Let me know if you need me to re-run an app-icon script I have on a 1024x1024 copy.